here-naksha-app-service/src/main/resources/static/openapi.yaml

```diff
@@ -247,6 +247,63 @@ paths:
           $ref: '#/components/responses/ErrorResponse504'
         '513':
           $ref: '#/components/responses/ErrorResponse513'
+  '/hub/spaces':
+    post:
+      tags:
+        - Manage Spaces
+      summary: Create new Space
+      description: Create new Space
+      operationId: postSpace
+      responses:
+        '201':
+          $ref: '#/components/responses/HandlerResponse'
```

Comment:
Was this meant to have HTTP status code 201 or 200? Also the response should be "SpaceResponse". That I have in my branch. Is it ok if I merge my branch here into this pull request?

Reply:
nice catch, fixed :)